groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyLanguageServer.java

```diff
@@ -40,7 +43,7 @@
 @SuppressWarnings("deprecation")
 public final class GroovyLanguageServer implements LanguageServer {
 
-    private static final Logger log = LoggerFactory.getLogger(GroovyLanguageServer.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(GroovyLanguageServer.class);
```

Comment:
I think for loggers the convention is to not user all uppercase naming because it is a constant reference and not a constant value. (http://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case)